console/module/job/src/main/java/org/eclipse/kapua/app/console/module/job/server/GwtJobTargetServiceImpl.java

```diff
@@ -112,18 +114,23 @@
             for (GwtJobTargetCreator gwtJobTargetCreator : gwtJobTargetCreatorList) {
                 if (findExtistingTarget(gwtJobTargetCreator.getJobTargetId(), existingTargets)) {
                     continue;
-                }
-                KapuaId creatorScopeId = KapuaEid.parseCompactId(gwtJobTargetCreator.getScopeId());
-                JobTargetCreator jobTargetCreator = JOB_TARGET_FACTORY.newCreator(creatorScopeId);
-                jobTargetCreator.setJobId(GwtKapuaCommonsModelConverter.convertKapuaId(gwtJobTargetCreator.getJobId()));
-                jobTargetCreator.setJobTargetId(GwtKapuaCommonsModelConverter.convertKapuaId(gwtJobTargetCreator.getJobTargetId()));
-
-                //
-                // Create the Job Target
-                JobTarget jobTarget = JOB_TARGET_SERVICE.create(jobTargetCreator);
-
-                // convert to GwtJobTarget and return
-                gwtJobTargetList.add(KapuaGwtJobModelConverter.convertJobTarget(jobTarget));
+                } 
+                    Device device = DEVICE_REGISTRY_SERVICE.find(KapuaEid.parseCompactId(gwtJobTargetCreator.getScopeId()), KapuaEid.parseCompactId(gwtJobTargetCreator.getJobTargetId()));
+                    if(device != null) {
+                         KapuaId creatorScopeId = KapuaEid.parseCompactId(gwtJobTargetCreator.getScopeId());
```

Comment:
Fix indentation. 

If you are using Eclipse IDE we have formatting and cleanup profiles under `dev-tools/src/main/eclipse/codeStyle/` 

`